index.js

```diff
@@ -122,3 +123,8 @@ Bankai.prototype.css = function (req, res) {
     return from([this._css])
   }
 }
+
+// (obj, obj) -> readStream
+Bankai.prototype.static = function (req, res) {
+  return send(req, req.url.substr(1));
```

Comment:
Fixed. Was fixed locally but I forgot to push the change because I have the following change in index.js too which seems necessary as soon as the url path is not `/` but which I don't want to add in this pull-request 😄 

```diff
diff --git a/index.js b/index.js
index 051e555..9168e87 100644
--- a/index.js
+++ b/index.js
@@ -42,8 +42,8 @@ function Bankai (entry, opts) {
 
   this._html = (function () {
     var base = {
-      script: 'bundle.js',
-      css: (self.cssDisabled) ? null : 'bundle.css',
+      script: '/bundle.js',
+      css: (self.cssDisabled) ? null : '/bundle.css',
       head: '<meta name="viewport" content="width=device-width, initial-scale=1">'
     }
     var html = createHtml(xtend(base, opts.html))
```